### PR TITLE
Expose stack on functions.yaml instead of stack.yaml.

### PR DESCRIPTION
**Pull request by taeold**
Some contractual changes to how we expose the container contract:

* We will expose `__/functions.yaml` instead of `__/stack.yaml` in the control api
* Environment variables for exposing the control api changes from `STACK_CONTROL_API_PORT` to `PORT` and `FUNCTIONS_CONTROL_API` (whose value should equal `true`).

### CODE REVIEW

File: scripts/bin-test/test.ts [hunk 1]
```diff
@@ -85,7 +85,8 @@ async function startBin(
     env: {
       PATH: process.env.PATH,
       GLCOUD_PROJECT: 'test-project',
-      STACK_CONTROL_API_PORT: port,
+      PORT: port,
```
[the remark this answers is not in the document]
Comment (inlined) [reply scripts/bin-test/test.ts:88]: The original design used two ports: one for the general API and one for the "control" API. We discovered that Python cannot handle two ports on the same process, so we are consolidating on one port. $PORT is the standard env variable for Cloud Run, so we're using it too.

File: src/bin/firebase-functions.ts [hunk 2]
```diff
@@ -35,21 +35,24 @@ async function handleQuitquitquit(req: express.Request, res: express.Response) {
...
-      .send(`Failed to generate manifest from function source: ${e}`);
-  }
-});
+
+if (process.env.FUNCTIONS_CONTROL_API === 'true') {
+  app.get('/__/functions.yaml', async (req, res) => {
+    try {
+      const stack = await loadStack(functionsDir);
+      res.setHeader('content-type', 'text/yaml');
+      res.send(JSON.stringify(stack));
```
Comment (inlined) [inline src/bin/firebase-functions.ts:44]: Lol. I guess JSON is technically YAML.

**Review by inlined — APPROVED** [review]

File: scripts/bin-test/test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaumbakahuma) [reply scripts/bin-test/test.ts:88]: Thank you for your service.

[the remark this answers is not in the document]
Comment (kaumbakahuma) [reply scripts/bin-test/test.ts:88]: Is ok

**Review by kaumbakahuma — COMMENTED** [review]
Let's you give us direction of to do us a community,we are one.

**Comment by kaumbakahuma** [comment]
I kaumbakahuma, I appreciate your help and support for this service from file Bass.
Let's continue to do so much for me.

**Pull request merged**